runner.go

```diff
@@ -3,24 +3,44 @@ package gfmxr
 import (
 	"fmt"
 	"io/ioutil"
+	"os"
 	"time"
 
 	"github.com/Sirupsen/logrus"
 )
 
 type Runner struct {
-	Sources []string
-	Count   int
-	Frobs   map[string]Frob
+	Sources   []string
+	Count     int
+	Frobs     map[string]Frob
+	Languages *Languages
 
 	log *logrus.Logger
 }
 
-func NewRunner(sources []string, count int, log *logrus.Logger) (*Runner, error) {
+func NewRunner(sources []string, count int, languagesYml string, log *logrus.Logger) (*Runner, error) {
+	if _, err := os.Stat(languagesYml); err != nil {
+		log.WithFields(logrus.Fields{
+			"url":  DefaultLanguagesYmlURL,
+			"dest": languagesYml,
+		}).Info("downloading")
+
+		err = PullLanguagesYml(DefaultLanguagesYmlURL, languagesYml)
```

Comment:
Might be nice to be able to disable this behavior in case the user just wants to use the explicit names and avoid pulling the linguist mapping explicitly or automatically.